exporters/trace/ocagent/src/main/java/io/opencensus/exporter/trace/ocagent/TraceProtoUtils.java

```diff
@@ -106,10 +105,9 @@ public AttributeValue apply(Double doubleValue) {
    *
    * @param spanData the {@code SpanData}.
    * @return proto representation of {@code Span}.
-   * @since 0.17
    */
   @SuppressWarnings("DefaultCharset")
```

Comment:
We should probably have reproducible builds so we need to use a charset here if that is needed.

Reply:
Removed - this seems to be I forgot to remove this annotation. It's no longer applicable after using `ByteString.copyFrom(bytes);`.